dsc/src/args.rs

```diff
@@ -24,6 +25,8 @@ pub struct Args {
     pub input: Option<String>,
     #[clap(short = 'p', long, help = "The path to a file used as input to the configuration or resource")]
     pub input_file: Option<String>,
+    #[clap(short = 'l', long = "logging-level", help = "Log level to display", value_enum, default_value = "Info")]
```

Comment:
Looks like this needs to be lowercase

```suggestion
    #[clap(short = 'l', long = "logging-level", help = "Log level to display", value_enum, default_value = "info")]
```

Reply:
Good point. Updated.